android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeInstance.kt

```diff
@@ -11,7 +21,90 @@ import io.customer.sdk.util.Logger
 object CustomerIOReactNativeInstance {
     internal val logger: Logger = ReactNativeConsoleLogger(CioLogLevel.ERROR)
 
-    fun setLogLevel(logLevel: CioLogLevel) {
+    fun setReactNativeLogLevel(logLevel: CioLogLevel) {
         (logger as ReactNativeConsoleLogger).logLevel = logLevel
     }
+
+    internal fun initializeSDKFromContext(context: Context) {
+        if (CustomerIO.instanceOrNull() != null) return
```

Comment:
I think we should let the user reinitialize because there might be a change in env variable or even config change and in that case, we wont let them do it?

Reply:
Yeah, makes sense for the case, updated 👍🏻